packages/generator-sprotty/package.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "generator-sprotty",
+    "version": "0.13.0",
+    "description": "Yeoman generator for Sprotty",
+    "engines": {
+      "node": ">=14.0.0"
+    },
+    "keywords": [
+      "yeoman-generator",
+      "diagram"
+    ],
+    "license": "(EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0)",
+    "files": [
+      "app",
+      "sprotty-local-template",
+      "src"
+    ],
+    "main": "app/index.js",
+    "types": "app/index.d.ts",
+    "scripts": {
+      "prepare": "yarn run clean && yarn run build",
+      "clean": "shx rm -rf app",
```

Comment:
We're currently using `rimraf` for that. It would be great to migrate to `shx`, but that should be done for all packages in a separate PR.